### ISSUE TITLE
[a11y]: FileUploaderItem has critical violation: The tabbable element's role 'img, graphics-document' is not a widget role

### ISSUE THREAD

**Issue by cvikram21** [carbon-design-system/carbon#13087]
### Package

carbon-components, carbon-components-react

### Browser

Firefox

### Operating System

MacOS

### Package version

7.57.4

### React version

17.0.2

### Automated testing tool and ruleset

IBM Equal Access Accessibility Checker: Latest Deployment

### Description

The FileUploaderItem when its status is complete has a critical a11y violation found as part of the testing of our product.
It can be reproducible in v10 storybook.

Screenshot from v10 storybook
<img width="1494" alt="image" src="https://user-images.githubusercontent.com/22382586/216668132-2dac4591-2e5d-4f8b-b25f-18445a203b1a.png">

### Reproduction/example

Carbon storybook: https://v10.carbondesignsystem.com/components/file-uploader/usage

### Steps to reproduce

1. On the v10 storybook for FileUploader, change the variant to upload states.
2. Select the status to complete.
3. Run the IBM Equal Access Accessibility Checker. The scan results will show the critical violation mentioned above.

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [X] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by sstrubberg** [comment]
Hey @cvikram21, thanks for drumming up the issue. Would you be open to making a repro against our [v10 storybook](https://v7-react.carbondesignsystem.com/?path=/story/getting-started--welcome)? Closing for now, thanks again!
